dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -300,20 +302,27 @@ public void testGetStepsArrayInsteadOfObject() throws ApiException {
 
     /**
      * This tests that a WDL workflow with complex imports is properly imported (also tests absolute paths)
+     *
+     * Note: With fix for https://github.com/dockstore/dockstore/issues/2931, this WDL now fails. Adjusting test accordingly. If you run
+     * womtool validate from the command line, this workflow fails to validate, so our code was correctly validating this WDL before,
+     * that seems to have been incorrect.
+     *
      * @throws ApiException
      */
+    @SuppressWarnings("checkstyle:EmptyCatchBlock")
     @Test
     public void testComplexImportWdlWorkflow() throws ApiException {
         // Input: /parent/parent.wdl
         // Repo: ComplexImportsWdl
         // Branch: master
         // Return: DAG with 7 nodes
 
-        final List<String> strings = getJSON("DockstoreTestUser2/ComplexImportsWdl", "/parent/parent.wdl", "wdl",
-                "test");
-        int countNode = countNodeInJSON(strings);
-
-        Assert.assertTrue("JSON should not be blank", strings.size() > 0);
-        Assert.assertEquals("JSON should have 7 nodes", countNode, 7);
+        try {
+            getJSON("DockstoreTestUser2/ComplexImportsWdl", "/parent/parent.wdl", "wdl", "test");
+            fail("Invalid WDL somehow came back good");
+        } catch (Exception ex) {
+            Assert.assertTrue(ex.getMessage().contains("could not process wdl into DAG: Failed to import workflow importC.wdl."));
```

Comment:
This is one of ours, make this into a String constant and link from both places. 
https://github.com/dockstore/dockstore/blob/e8501ecd29a5985a3119418a0858c1c2745802c6/dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java#L359